### DISCUSSION TITLE
Syncing via onedrive only one way

### DISCUSSION THREAD

**Discussion by Lanalangz** [remotely-save/remotely-save#311] [Q&A]
Hi there
I installed the plugin and set it up via dropbox and it worked fine on my ipad, but when trying to set it up on my phone i realized I didn't have more dropbox device usages, so I decided to switch to onedrive, but when setting up via onedrive I am only  able to get files from my pc to my ipad, not from ipad to pc. 
Is this not possible with onedrive?

**Comment by fyears** [comment]
i believe the plugin should work universally
